doc/reference/devices_nic.md

```diff
@@ -103,6 +103,21 @@ Key                      | Type    | Default           | Managed | Description
 `vlan`                   | integer | -                 | no      | The VLAN ID to use for non-tagged traffic (can be `none` to remove port from default VLAN)
 `vlan.tagged`            | integer | -                 | no      | Comma-delimited list of VLAN IDs or VLAN ranges to join for tagged traffic
 
+#### Configuration examples
+
+Add a `bridged` network device to an instance using `nictype`:
+
+    lxc config device add <instance_name> <device_name> nic nictype=bridged parent=<existing_bridge>
+
+Add a `bridged` network device to an instance using a managed network:
+
+    lxc network create <network_name> --type=bridge
+    lxc config device add <instance_name> <device_name> nic network=<network_name>
+
+Note that the network type is `bridged` when using `nictype`, but `bridge` when using a managed network.
```

Comment:
This is confusing IMHO. The nictype for the NIC in both cases is `bridged`, its just that when specifying a managed network using `network` LXD can infer the NIC type from the network type.

Reply:
Reworded. I think it's important to point this out, because for all other networks, the `nictype` value and the value for the type of the managed network are the same.